tools/devops/automation/scripts/bash/clean-bot.sh

```diff
@@ -12,6 +12,14 @@ df -h
 # We don't care about errors in this section, we just want to clean as much as possible
 set +e
 
+# Clean workspace
+(
+	if test -d "$SYSTEM_DEFAULTWORKINGDIRECTORY/xamarin-macios"; then
```

Comment:
@rolfbjarne  Can you make the name of the repo a variable somewhere central? Maybe first line in the top level Make.config? Some sed'ing may be involved heh but since we are migrating the repo over we can start adding to the machinery to no depend on the magic string `xamarin-macios`

Reply:
@dalexsoto fixed!